.envrc

```diff
@@ -5,17 +5,11 @@ missing() {
 
 set -e
 
-if missing pre-commit; then
-    echo >&2 "You're missing pre-commit. Install it with brew install pre-commit"
-    return 1
-fi
-# If installed, it executes very fast
-pre-commit install >/dev/null
```

Comment:
we want everyone to use the git hooks

Reply:
@asottile-sentry is `brew install pre-commit` the preferred approach?
I would have expected devenv in the sentry's checkout to install it for me.

Reply:
oh I thought this was sentry uhmmm hmmmm

Reply:
I'm not quite sure then

Reply:
Can we remove it for this project?

Reply:
installing it with brew I think is fine -- I would keep this block